Woof/WoofTests/Model/Preferences/PreferencesHandlerLoadTests.swift

```diff
@@ -0,0 +1,50 @@
+import XCTest
+
+final class PreferencesHandlerLoadTests: XCTestCase {
+    private var userPreferencesStorage = KeyValueStorage.preferencesStorage
+
+    override func setUp() {
+        super.setUp()
+        userPreferencesStorage.removeAll()
+    }
+
+    override func tearDown() {
+        userPreferencesStorage.removeAll()
+        super.tearDown()
+    }
+
+    func testLoadSelectedRoleExistsInAPI() {
```

Comment:
the name of the test doesn't match the method name, we don't have method `loadSelectedRole`. Please review other tests.